registration/views.py

```diff
@@ -23,6 +29,31 @@ def get_serializer_class(self):
             return FamilyDetailSerializer
         return FamilySerializer
 
+    @action(
+        methods=["get"],
+        detail=False,
+        permission_classes=[permissions.IsAuthenticated],
+        url_path="search",
+        url_name="search",
+    )
+    def search(self, request):
+        first_name = self.request.query_params.get("first_name")
+        last_name = self.request.query_params.get("last_name")
+        if not first_name and not last_name:
+            return Response(
+                "No search parameters found.", status=status.HTTP_400_BAD_REQUEST
+            )
+        result = Family.objects
```

Comment:
this won't run a db query right? I'm guessing `Family.objects.all()` would run a query